### PR TITLE
Added context menu option in electron for all input fields AB#16922

### PR DESCRIPTION
**Pull request by wbreza**
- Adds right-click context menu options for all input elements in electron app
image

### CODE REVIEW

File: src/electron/main.ts [hunk 1]
```diff
@@ -59,6 +62,42 @@ function onToggleDevTools(sender: any, show: boolean) {
     }
 }
 
+/**
+ * Adds standard cut/copy/paste/etc context menu comments when right clicking input elements
+ * @param browserWindow The browser window to apply the context-menu items
+ */
+function registerContextMenu(browserWindow: BrowserWindow): void {
```
Comment (wbreza) [inline src/electron/main.ts]: I don't have a good way to test this... any ideas?

File: src/electron/main.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import { app, ipcMain, BrowserWindow, dialog, BrowserWindowConstructorOptions } from "electron";
+import { app, ipcMain, BrowserWindow, dialog, BrowserWindowConstructorOptions, Menu } from "electron";
 import { IpcMainProxy } from "./common/ipcMainProxy";
 import LocalFileSystem from "./providers/storage/localFileSystem";
+import { register } from "../serviceWorker";
 
-// Keep a global reference of the window object, if you don't, the window will
+// Keep a global reference of the window object, if you don"t, the window will
```
Comment (mydiemho) [inline src/electron/main.ts]: Nit: typo

**Comment by wbreza** [comment]
Fixes #562

File: src/electron/main.ts (hunk 1 above)
Comment (JacopoMangiavacchi) [reply src/electron/main.ts]: No idea sorry

Comment (PIC123) [reply src/electron/main.ts]: I don't know if snapshot testing would even show anything here so I'm not sure either.

**Pull request merged**